src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -227,9 +237,11 @@ private Case buildCaseFromSourceCaseAndEvent(CollectionCase newCollectionCase, C
     newCase.setOa(sourceCase.getOa());
     newCase.setPrintBatch(sourceCase.getPrintBatch());
     newCase.setSurvey(sourceCase.getSurvey());
+    newCase.setMetadata(metadataFromSourceCase(sourceCase.getMetadata()));
```

Comment:
I thought we agreed we weren't going to do this - it should be explicitly copying the `secureEstablishment` value into a brand new `CaseMetadata` object.